magenta/tools/magenta-install.sh

```diff
@@ -85,7 +85,7 @@ echo "setting up magenta environment"
 echo "=============================="
 echo ""
 
-conda create -n magenta python=2.7
+conda create -n magenta python=3.6
```

Comment:
should probably also change the download above to get Miniconda3 instead of Miniconda2.

Reply:
Done. Also, pip changed to pip3.

Reply:
one other nit: can you change the path for CONDA_PREFIX to also use miniconda3?

Reply:
Done